### PR TITLE
Complete version

### PR DESCRIPTION
**Pull request by maen-al-khraisha**
Application feature:
- show the COVID cases all around the world with the total cases for each continent.
- when clicking on the continent you will see a list of countries inside that continent.
- inside the country list you can filter according to the country name.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,84 @@
...
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- React / Redux
+- JS
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
 
-### `npm test`
+## Technologies
```
Comment (rustam-tolipov) [inline README.md:15]: [OPTIONAL] Kindly add a Screenshot to your project.

File: README.md [hunk 2]
```diff
@@ -1,46 +1,84 @@
...
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Design is inspired from a design made by [Nelson Sakwa](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)).
+- Linters are made by [Microverse Inc](https://github.com/microverseinc/linters-config/).
+
+## 📝 License
+
+This project is released under [AGPL](./LICENSE) licensed.
```
Comment (rustam-tolipov) [inline README.md]: [OPTIONAL] Please add the MIT license to your project.

File: src/features/COVID/COVID-API.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+// A mock function to mimic making an async request for data
```
Comment (rustam-tolipov) [inline src/features/COVID/COVID-API.js:1]: - [ ] App has unit tests for several components of the app and some pure functions in Redux code. You only made an integration test for your project. Kindly add unite tests as well.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @maenkhraisha, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elyor-doniyorov — APPROVED** [review]
Hi @maenkhraisha, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Implemented requested changes

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**